tla/consensus/MCccfraft.cfg

```diff
@@ -59,6 +53,9 @@ CHECK_DEADLOCK
 POSTCONDITION 
     WriteStatsFile
 
+_PERIODIC
+    SerializeCoverage
```

Comment:
What does that do?

Reply:
The [_periodic hook](https://github.com/tlaplus/tlaplus/commit/23f765078cd098a2bc2bc3d19889ba0c64583ca4) is a provisional feature that periodically evaluates some constant-level expression.  In our case, `SerializeCoverage`  serializes TLC's coverage information.

Reply:
* I did not introduce _PERIODIC in this PR
* I don't see any issue with removing _PERIODIC. IIRC, we added it when you were debugging the state-space explosion in the consistency spec
* The risk of unexpected removal is not a concern because, if anyone were to remove _PERIODIC, it would be me